public/javascript/admin.js

```diff
@@ -127,14 +127,20 @@ function updateUserOverview() {
         iconText[0] = iconText[0].toUp
         userTypingHidden = chat.typing ? '' : 'hidden';
         messagePreview = chat.messages.length == 0 ? '' : chat.messages[chat.messages.length - 1].message;
+        alert = chat.alert ? " id=alert" : '';
         tab.innerHTML = tab.innerHTML 
                       + "<button class='username' " + selectedChat
                       + " onclick='toggleChat(`" + chat.userId + "`)'>"
-                      + iconTag
-                      + "<div class='buttonText'><div class='buttonId'>" + iconText + "</div>"
-                      + "<div class='messagePreview'>" + messagePreview + "</div></div>"
-                      + "<div class='buttonTypingDiv' " + userTypingHidden + ">"
-                      + "<img class='buttonTypingIcon' src='img/typing_icon.png'></div></button>";
+                        + iconTag
```

Comment:
It looks like there's some mixing of tabs and spaces here, can you change them to all be spaces?

Reply:
was on purpose to make the nesting structure clearer but I can change it back to all left aligned '+' if you want

Reply:
(theyre all spaces)

Reply:
Oh ok, that makes sense